### PR TITLE
Add sama network with icp showcase

### PR DESCRIPTION
**Pull request by gaemltim**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Comment by sa-github-api** [comment]
Dear @gaemltim,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/369) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: showcase.json [hunk 1]
```diff
@@ -2811,6 +2811,26 @@
     "youtube": "https://www.youtube.com/channel/UCdMLfGmuLAsQlVPg9QROfcQ",
     "submittableId": "32119235"
   },
+  {
+    "name": "Sama Network",
+    "description": "All data incurred from our daily internet usage are being hijacked, monitored and analyzed by centralized companies.Traditional blockchain systems often construct on-chain assets and contract data for privacy, yet lack of effective privacy and security protection for specific business data. Meanwhile, the emergence of AI and the explosive growth of computation power demand by small and medium AI models in research and commercialization, the trend of anti-globalization has also brought about the division of computational power. That’s when  platforms like BOINC shines as it shares the dividends brought about by the development of the rising distributed computing market. SAMA network, with its original designed consensus and business sharding separation, builds a highly secure and private distributed computational system. It provides data security and computational power for the World Wide Web without altering users’ habits.",
```
Comment (jennifertrin) [inline showcase.json:2816]: The description is quite long. Can you please limit the description to max two-sentence tagline?

**Review by jennifertrin — CHANGES_REQUESTED** [review]
Can you please review the note above?

File: showcase.json (hunk 1 above)
Comment (gaemltim) [reply showcase.json:2816]: OK

File: showcase.json [hunk 2]
```diff
@@ -2811,6 +2811,26 @@
     "youtube": "https://www.youtube.com/channel/UCdMLfGmuLAsQlVPg9QROfcQ",
     "submittableId": "32119235"
   },
+  {
+    "name": "Sama Network",
+    "description": "SAMA network, with its original designed consensus and business sharding separation, builds a highly secure and private distributed computational system. It provides data security and computational power for the World Wide Web without altering users’ habits.",
+    "website": "https://sama.network/",
+    "logo": "/img/showcase/sama_network_logo.png",
+    "screenshots": [],
+    "video": "",
```
Comment (jennifertrin) [inline showcase.json:2820]: Can you please remove the video key value if you do not have a video?

File: showcase.json [hunk 3]
```diff
@@ -2811,6 +2811,26 @@
     "youtube": "https://www.youtube.com/channel/UCdMLfGmuLAsQlVPg9QROfcQ",
     "submittableId": "32119235"
   },
+  {
+    "name": "Sama Network",
+    "description": "SAMA network, with its original designed consensus and business sharding separation, builds a highly secure and private distributed computational system. It provides data security and computational power for the World Wide Web without altering users’ habits.",
+    "website": "https://sama.network/",
+    "logo": "/img/showcase/sama_network_logo.png",
+    "screenshots": [],
```
Comment (jennifertrin) [inline showcase.json:2819]: Can you please remove the screenshot key value if you do not have a video?

File: showcase.json (hunk 2 above)
Comment (gaemltim) [reply showcase.json]: OK,got it.

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**